src/providers/ldap/sdap_id_op.c

```diff
@@ -621,6 +749,17 @@ static void sdap_id_op_connect_done(struct tevent_req *subreq)
                   ret, sss_strerror(ret));
             /* Avoid causing the whole backend to be marked as offline because
              * this operation failed. */
+            /* Commented out because Coverity detects that ret is overwritten
+             * immediately after this. */
+            /*ret = EOK;*/
```

Comment:
Hi,

sorry, I missed that, since `ret` here is not the final error code, please remove all the comments above from here.

bye,
Sumit